go.mod

```diff
@@ -14,7 +14,7 @@ replace (
 require (
 	github.com/ajeddeloh/go-json v0.0.0-20170920214419-6a2fe990e083 // indirect
 	github.com/coreos/ignition v0.33.0
-	github.com/go-logr/zapr v0.1.0
+	github.com/go-logr/zapr v0.1.1
```

Comment:
Why are you changing the version in this PR? Changing the version warrants its own PR. Or if it is needed the reason needs to be clearly called out in the commit message.

Reply:
I am not intentionally doing this. We might want to ping all library versions.